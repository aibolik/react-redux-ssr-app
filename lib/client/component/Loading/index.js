import React from 'react';

const Loading = props => React.createElement(
  'h4',
  null,
  'Loading...'
);

export default Loading;