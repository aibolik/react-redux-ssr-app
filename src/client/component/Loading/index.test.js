import React from 'react';
import Loading from './index';
import renderer from 'react-test-renderer';

describe('Test Loading component', () => {

  it('renders correctly', () => {
    const tree = renderer.create(<Loading />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
