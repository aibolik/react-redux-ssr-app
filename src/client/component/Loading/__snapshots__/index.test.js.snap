// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Loading component renders correctly 1`] = `
<h4>
  Loading...
</h4>
`;
